lms/static/js/student_profile/views/learner_profile_fields.js

```diff
@@ -16,22 +17,29 @@
             },
 
             showNotificationMessage: function () {
-                var accountSettingsLink = '<a href="' + this.options.accountSettingsPageUrl + '">' + gettext('Account Settings page.') + '</a>';
+                var accountSettingsLink = StringUtils.interpolate('<a href="{url}">Account Settings page.</a>',
```

Comment:
I tried using `joinHtml()` and ending up with something like:
```
HtmlUtils.joinHtml(
  HtmlUtils.HTML('<a href="),
  this.options.accountSettingsPageUrl,
  HtmlUtils.HTML(' ">'),
  gettext('Account Settings page.'),
  HtmlUtils.HTML('</span>')
)
```
but it didn't work and looked messy. Any suggestions?